src/App.tsx

```diff
@@ -18,10 +20,38 @@ import NotFound from "./UI/pages/NotFound";
 
 const App: FC = () => (
   <Routes>
-    <Route path={ROUTES.HOME} element={<Home />} />
-    <Route path={ROUTES.MOVIES} element={<Movies />} />
-    <Route path={ROUTES.TRENDING} element={<Trending />} />
-    <Route path={ROUTES.MOVIE} element={<Movie />} />
+    <Route
+      path={ROUTES.HOME}
+      element={
+        <AuthLayout>
+          <Home />
+        </AuthLayout>
+      }
+    />
+    <Route
```

Comment:
create one
   <Route
            path="app"
            element={(
              <AuthRoute>
                <AuthLayout />
              </AuthRoute>
            )}
          >
          
          as a wrapper and wrap all other routes